lidar_multiclass/callbacks/comet_callbacks.py

```diff
@@ -50,19 +55,21 @@ def __init__(self, code_dir: str):
     @rank_zero_only
     def on_train_start(self, trainer, pl_module):
         logger = get_comet_logger(trainer=trainer)
-        experiment = logger.experiment
-
-        for path in Path(self.code_dir).resolve().rglob("*.py"):
-            experiment.log_code(file_name=str(path))
-        log.info("Logging all .py files to Comet.ml!")
+        if logger:
```

Comment:
c'est sans doute une question qui se discute, mais j'ai une petite préférence pour :
        if logger:
            for path in Path(self.code_dir).resolve().rglob("*.py"):
                logger.experiment.log_code(file_name=str(path))

C'est une remarque que j'aurais pu mettre à d'autres endroits, souvent tu instantialises une variable pour qu'elle ne soit utilisée qu'une fois, par exemple 
variable = f(x)
return variable
au lieu de : 
return f(x)